CICD/github/launch_karate_tests.sh

```
@@ -16,12 +16,12 @@ export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
 [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
 source ./bin/load_environment_light.sh;
-cd config/docker
-./startOpfabInProductionMode.sh
-cd ../../src/test/api/karate
+cd bin
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)